src/BlogPost.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+
+// props: {
+//   post: {
+//     id: number,
+//     title: string,
+//     author: string,
+//     createdOn: string,
+//     body: string,
+//     tags: arrayOf(string),
+//   }
+// }
+
+export class BlogPost extends Component {
+  render() {
+    const { post } = this.props;
+
+    function renderDate(date) {
+      const datePieces = date.split("-");
+      const [year, month, day] = datePieces;
+      return `Publish Date: ${month} - ${day} - ${year}`;
+    }
+
+    return (
+      <div className="post-container">
+        <h2 className="post-title">{post.title}</h2>
+
+        <div className="author-container">
+          Publised By:
+          <a href="#" className="post-author">{`${post.author}`}</a>
```

Comment:
you don't need the template string here. you're not adding anything to the `post.author`, so this can be rendered the same way as `post.title` was